src/sql/magic.py

```diff
@@ -260,10 +260,27 @@ def execute(self, line="", cell="", local_ns=None):
 
     @telemetry.log_call("execute", payload=True)
     def _execute(self, payload, line, cell, local_ns, is_interactive_mode=False):
+        # Remove trailing semicolons
```

Comment:
let's move this logic inside SQLStore since it makes more sense to encapsulate it there. more precisely, let's remove the `;` only when rendering the final result: 

https://github.com/ploomber/jupysql/blob/bfc1c6c3e8833da047a7840b57f35e76ca6e40f8/src/sql/store.py#L101

I think you can just copy paste your logic

Reply:
got it! i included the logic in `SQLQuery` -> `__str__` as well, otherwise the last query which gets appended onto the templates includes a semicolon